### ISSUE TITLE
Predictable and fast release schedule for ocpp library

### ISSUE THREAD

**Issue by villekr** [mobilityhouse/ocpp#238]
This is a process issue.

We have made a decision to rely on this library and even invested efforts to test and improve the library. Now the situation is such that latest release in pypi is 0.8.3 from 21.4.2021. There's a few issues pending on the new release but no information about expected timeframe. Would it be possible to try set some kind of rough promise and timeframe how long it takes for a PR to go trough the review and end up released.

**Comment by laysauchoa** [comment]
It's a good question, @villekr. Maybe @OrangeTux can help us later with that. However, many people are on vacation at this moment.

**Comment by OrangeTux** [comment]
Thanks for your comment @villekr . Currently the release process is manual and depending on me. That is not a good thing. and we this process can be improved. I've created a ticket to automate this process, see #245. 

I also will give @tropxy full permissions to this project. So multiple people can create releases.

Promises I won't give. This is an open source project and I maintain  this project on a best effort basis. 

Btw, I just released v0.9.0 of this project.
